frontend/public/style/_common.scss

```diff
@@ -137,7 +137,12 @@
 
 .co-m-pane__explanation {
   color: $color-grey-text;
-  margin-bottom: 20px;
+  margin-bottom: 30px;
+}
+
+.co-m-pane__explanation code {
+  background-color: $co-gray-lightest;
+  color: $co-gray;
```

Comment:
@jennyhaines Do we want to change the `code` style in only one spot?

I wonder if we should just not use `<code>` tags if we don't want the styling.

Reply:
@spadgett @rebeccaalpert I actually don't see the need to have the code styling be so prominent with the blue anywhere. It seems too prominent within a block of text that is generally used as description text. Since this text is not meant to be more important than (or detract from) its subsequent form field or data display, I'd suggest changing the styling to grey everywhere rather than blue. I was looking at [this example](http://www.patternfly.org/pattern-library/widgets/#cnt-remaining-chars) on PatternFly of the `<code>` tag being used multiple times in one block, and it just seems like too much.